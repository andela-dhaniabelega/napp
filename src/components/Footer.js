import React from 'react';
import '../../sass/styles.scss';

export default class Featured extends React.Component {
	render() {
		return(
			<div className="footer">
				<footer>
					<p>2017 Napp Ltd. All Rights Reserved.</p>
					
				</footer>
			</div>
      );
	}
}
