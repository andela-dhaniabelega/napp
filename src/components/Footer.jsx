import React from 'react';
import '../../public/sass/styles.scss';


/**
 * Stateless component
 * Simply renders Footer for components
 */
const Footer = () => (
  <div className="footer_bottom_area">
    <div className="copyright_text">
      <p>Copyright &copy; 2017 Napp. All rights reserved</p>
    </div>
  </div>

);

export default Footer;








    








