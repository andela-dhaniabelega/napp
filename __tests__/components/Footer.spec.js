import React from 'react';
import renderer from 'react-test-renderer'
import Footer from '../../src/components/Footer';


describe('Footer component', () => {
		it('renders a snapshot', () => {
          const tree = renderer.create(<Footer />).toJSON();
          expect(tree).toMatchSnapshot();
		});
});