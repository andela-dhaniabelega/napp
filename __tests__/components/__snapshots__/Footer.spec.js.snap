// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Us Page renders a snapshot 1`] = `
<div
  className="footer_bottom_area"
>
  <div
    className="copyright_text"
  >
    <p>
      Copyright Â© 2017 Napp. All rights reserved
    </p>
  </div>
</div>
`;
